doc/rst/platforms/macosx.rst

```diff
@@ -4,40 +4,57 @@
 Using Chapel on Mac OS X
 ========================
 
-By and large, Chapel can be used on a Macintosh running OS X just like
-any other platform -- refer to the :ref:`chapelhome-quickstart` for more
-information on building and using Chapel.
+There are two main approaches for using Chapel on Mac OS X:
+
+1) Install via Homebrew_.  For existing Homebrew users, this is the
+   quickest way to get up and running, but it results in a copy of
+   Chapel that only supports shared-memory (single-locale) executions.
+
+2) Install and build Chapel from source, as with any other UN*X
+   system.  This is slightly more involved, but supports Chapel's full
+   feature set.  Refer to the :ref:`chapelhome-quickstart` for more
+   information.
 
 --------
 Homebrew
 --------
 
 A minimal installation of Chapel can be obtained through Homebrew_ with the
-following command::
+following commands::
 
+    brew update
     brew install chapel
 
-The above command installs a release version of Chapel. It is also
-possible to use Homebrew_ to build and install a development version
-based on the master branch on GitHub::
+These commands install the latest release of Chapel.  When using a
+Homebrew installation of Chapel, the ``CHPL_HOME`` directory is here::
 
-    brew install chapel --HEAD
+    /usr/local/Cellar/chapel/<chapel-version>/libexec/
 
-When using a Homebrew install of Chapel, the ``CHPL_HOME`` directory is
-here::
+Compile and run a test program::
 
-    /usr/local/Cellar/chapel/<chapel-version>/libexec/
+    chpl /usr/local/Cellar/chapel/<chapel-version>/libexec/hello.chpl
+    ./hello
+
+If you're new to Chapel, refer to the `What's Next?
+<https://chapel-lang.org/docs/usingchapel/QUICKSTART.html#what-s-next>`_
+section of :ref:`chapelhome-quickstart` for next steps.
+    
+.. note::
```

Comment:
This is a bit repetitive, but if you think it is important to emphasize, it is fine

Reply:
Repetitive in that it links to the Quickstart doc and the what's next section of the Quickstart doc you mean?

Reply:
I actually meant it repeats what we said in the beginning of this doc. i.e. that you don't get multilocale Chapel if you install via `brew`.

Reply:
Oh sorry for my confusion... GitHub's view of the excerpt you commented on emphasized the previous sentences, not the note you were referring to.  I understand now.  I think having it in a bold note covers our butts a bit more (I feel like I still hear people express surprise that their homebrew doesn't support multi-locale... not that this raises the chance they'll read it by much, but...).  Thanks!

Reply:
Yeah that makes sense.